workers/unity/Assets/Gdk/Core/Error/ConnectionError.cs

```diff
@@ -0,0 +1,12 @@
+namespace Improbable.Gdk.Core
+{
+    public struct ConnectionError
+    {
+        public readonly string Message;
```

Comment:
The point of this object is to help a user write retry logic. As a rule of thumb a string is human readable, enums etc are machine readable. So a string is needed for you to be able to log things in the case of failure. But you need some other type of error data for this to be useful.

Reply:
Added an enum field.